gunicorn/glogging.py

```diff
@@ -199,7 +199,9 @@ def setup(self, cfg):
                 self.access_log, cfg, self.syslog_fmt, "access"
             )
 
-        if cfg.logconfig:
+        if cfg.logconfigdict:
+            dictConfig(cfg.logconfigdict)
```

Comment:
Haven't played with the patch yet, but perhaps we should update CONFIG_DEFAULTS first and then pass it to dictConfig?